packages/core/contracts-ovm/insured-bridge/implementation/OVM_BridgeDepositBox.sol

```diff
@@ -211,8 +210,8 @@ contract OVM_BridgeDepositBox is OVM_CrossDomainEnabled, OVM_Testable {
         );
 
         emit FundsDeposited(
+            69, // In Solidity v8 can use chainId: https://docs.soliditylang.org/en/v0.8.0/units-and-global-variables.html
```

Comment:
nice

Reply:
lol. 69 is actually network details for OVM-kovan, 10 for mainnet. Do you think we should parameterize this or keep it hardcoded? https://community.optimism.io/docs/infra/networks.html